### PR TITLE
feat(uirefresh): component refresh - chart header

### PR DESCRIPTION
**Pull request by moo-onthelawn**
Small revamps to the chart header + polishes. [Figma](https://www.figma.com/design/fMQodZKzn9B5aZTN5G0fLJ/dYdX-%E2%80%BA-Web?node-id=38229-12578&node-type=canvas&t=XzATpZKvqySt2e9I-0). Flagged behind `uirefresh=true`

Notable things I did not do here:
- Did not update tab styling (so that we have no dividers, just accented underlines underneath). That'll come in a follow up.

---

## Views

* `MarketSelectorAndStats`
  * Add fade to end to make it more obvious it's scrollable

Screenshot 2024-09-27 at 2 36 14 PM

* `MarketLinks`
  * Remove border and background when `uirefresh` enabled

  
| before | after | 
| ---- | ---- |
| Screenshot 2024-09-27 at 2 37 14 PM | Screenshot 2024-09-27 at 2 37 30 PM |

* `MarketsDropdown`
  * Make asset icon bigger when `uirefresh` enabled
  * Made asset icons in dropdown bigger when `uirefresh` enabled
  
Screenshot 2024-09-30 at 3 54 27 PM

## Components

* `AssetIcon`
  * Adds configurable icon height/width

* `icons/website.svg`
  * Re-exported a version with less white space on the outside

## Styles/Mixins

* `styles/constants.css`
  * Update market dropdown width when `uirefresh` enabled

* `styles/layoutMixins.css`
  * Split `scrollAreaFade` into `start` and `end`

---

### CODE REVIEW

File: src/components/AssetIcon.tsx [hunk 1]
```diff
@@ -784,19 +784,25 @@ export const AssetIcon = ({
...
+`;
+
 const $Placeholder = styled.div`
+  --asset-icon-height: 1em;
+
   background-color: var(--color-layer-5);
-  width: 1em;
-  height: 1em;
+  width: var(--asset-icon-height);
+  height: var(--asset-icon-height);
```
Comment (jaredvu) [inline src/components/AssetIcon.tsx:805]: I imagine theres a couple other places that rely on `font-size` to adjust the width/height. Were you able to audit all the usages to use `--asset-icon-height` instead?

Comment (jaredvu) [reply src/components/AssetIcon.tsx:805]: maybe we should also slap a `min-width`/`min-height` as well?

Comment (moo-onthelawn) [reply src/components/AssetIcon.tsx:805]: hmm `--asset-icon-height` defaults to `1em` on line 801, so if I'm understanding correctly, this shouldn't actually impact other existing usages should it? 👀 (or maybe I'm misunderstanding) - but def agree on min-width/height! will add

Comment (moo-onthelawn) [reply src/components/AssetIcon.tsx]: ah actually did u mean this [fix](https://github.com/dydxprotocol/v4-web/pull/1098) I ended up having to do 😢

**Review by tyleroooo — APPROVED** [review]

**Pull request merged**